### DISCUSSION TITLE
Transform label using getOptionLabelUsing

### DISCUSSION THREAD

**Discussion by oliverbj** [filamentphp/filament#5811] [Help] [answered]
Hi there.

I have this Form field:

```php
Select::make('booking.consignee')
        ->placeholder('Select a consignee')
        ->label('Consignee')
        ->searchable()
        ->relationship('consignee', 'name')
        ->getOptionLabelUsing(fn (string $label) => strtolower($label)),
```

I would assume that I can transform the label of the select using the `getOptionLabelUsing` method - in my case, I want to lowercase the label (it comes in uppercase from my DB).

Am I missing anything here?

**Comment by danharrin** [comment, marked as answer]
Try `getOptionLabelFromRecordUsing(fn ($record) => strtolower($record->name))`

`getOptionLabelUsing()` is something different which is used to transform a value stored in the db into a label

**Reply by oliverbj** [reply]
That did the trick - thank you, Dan!
